typescript-conditionals/main.ts

```diff
@@ -70,17 +70,14 @@ const homer: Person = {
 };
 console.log('value of isOldEnoughToDrive: ', isOldEnoughToDrive(homer));
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Do not add `eslint` rules without asking me. We have our `eslint` setup a specific way, so if you are needing to add a rule to bypass an error, then there is something that is going wrong. If you are getting an error, then please reach out and ask, but don't add any rules yourself.

Reply:
got it, thanks